src/controllers/user.controller.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-shopping
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {repository} from '@loopback/repository';
+import {post, param, get, requestBody, HttpErrors} from '@loopback/rest';
+import {User} from '../models';
+import {UserRepository} from '../repositories';
+import {genSalt, hash} from 'bcrypt';
+import {promisify} from 'util';
+import * as isemail from 'isemail';
+
+const genSaltAsync = promisify(genSalt);
+const hashAsync = promisify(hash);
+
+export class UserController {
+  constructor(
+    @repository(UserRepository) public userRepository: UserRepository,
+  ) {}
+
+  @post('/users')
+  async create(@requestBody() obj: User): Promise<User> {
```

Comment:
nitpick: I suggest we rename `obj` to `userObj` or `user` or something along those lines

Reply:
That's the default we generate using the `lb4 controller` command but a good call. Will update.